packages/components/src/components/EDSProvider/EDSProvider.tsx

```diff
@@ -23,12 +25,15 @@ export const EDSProvider = (
             value={{ colorScheme: props.colorScheme, density: props.density }}>
             <GestureHandlerRootView style={{ flex: 1 }}>
                 <PortalProvider>
-                    <Portal.Host style={{ flex: 1 }} name="root">
-                        {props.children}
-                    </Portal.Host>
+                    <ScrimProvider>
```

Comment:
The naming of this provider makes it seem like it is different to the `PortalProvider`.
Maybe rename this to `ScrimPortal`to better communicate its intent (?)

Reply:
I don't think how the provider works needs to be reflected in the name